packages/graphback/src/InputTypeVisitor.ts

```diff
@@ -1,7 +1,16 @@
-import { DirectiveDefinitionNode, FieldDefinitionNode, ListTypeNode, NamedTypeNode, NameNode, NonNullTypeNode, ObjectTypeDefinitionNode } from 'graphql';
+import { ArgumentNode, BooleanValueNode, DirectiveDefinitionNode, DirectiveNode, FieldDefinitionNode, ListTypeNode, NamedTypeNode, NameNode, NonNullTypeNode, ObjectTypeDefinitionNode, StringValueNode } from 'graphql';
 
 const scalars = ['ID', 'Int', 'Float', 'String', 'Boolean']
 
+const defaultConfig = {
```

Comment:
Quick question. Do we still use directives here and override this values?

Reply:
Yes we do that while visiting the nodes.